pages/workspace/connect-inspector-to-posthog.mdx

```diff
@@ -0,0 +1,116 @@
+---
+layout: 'guide'
+title: 'Send data from PostHog to Avo Inspector'
+---
+
+import Link from '../../components/Link';
+
+## Use Avo Inspector as a PostHog destination
+
+If you are already sending your data to PostHog you can stream it to Avo Inspector without adding any code to your codebase.
+
+PostHog has an official plugin maintained by PostHog that sends data directly to the Avo Inspector. If you are using Posthog you can set it up within a few minutes.
+
+> The Plugin is a side effect of the Avo Inspector analyzing your events and keeping your PostHog events stream unchanged.
+
+After connecting the PostHog plugin to a source the event schemas of all events seen in PostHog will automatically be sent to Avo Inspector and you'll be able to monitor the quality of your tracking implementation without any changes in your code!
+
+> Remember that production data takes up to 2 hours to appear in the Inspector dashboard, while development data is available without any delay
```

Comment:
Should we also highlight that Inspector only receives the event schemas, and not the actual data?